we currently use the bundled 2geom instead of a system copy, so don't install
the actual library dev files
--
--- a/src/3rdparty/CMakeLists.txt
+++ b/src/3rdparty/CMakeLists.txt
@@ -6,7 +6,7 @@
 
 if(WITH_INTERNAL_2GEOM)
     set(2GEOM_BUILD_SHARED ${BUILD_SHARED_LIBS} CACHE BOOL "")
-    add_subdirectory(2geom)
+    add_subdirectory(2geom EXCLUDE_FROM_ALL)
 endif()
 
 if(WITH_INTERNAL_CAIRO)
